packages/@glimmerx/storybook/.eslintrc.js

```diff
@@ -0,0 +1,258 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
```

Comment:
we should be able to just use the babel-eslint parser now since it has the typescript preset

Reply:
We discussed we will stick with typescript-eslint/parser since babel-eslint parser yields different lint rules